sonar-php-plugin/src/main/java/org/sonar/plugins/php/PHPSensor.java

```diff
@@ -140,13 +139,10 @@ private static boolean inSonarQube(SensorContext context) {
   }
 
   private void processCoverage(Map<String, Integer> numberOfLinesOfCode, SensorContext context) {
-    PhpUnitService phpUnitSensor = new PhpUnitService(
-      fileSystem,
-      new PhpUnitTestResultImporter(),
-      new PhpUnitCoverageResultImporter(fileSystem),
-      new PhpUnitItCoverageResultImporter(fileSystem),
-      new PhpUnitOverallCoverageResultImporter(fileSystem));
-    phpUnitSensor.execute(context, numberOfLinesOfCode);
+    new PhpUnitTestResultImporter().parseReport(context, numberOfLinesOfCode);
```

Comment:
Renamed everything to importReport. Have a look at the latest solution, then we see if we go for the separate interfaces and drop the numberOfLinesOfCode from the unit test results